### PR TITLE
feat: add information form

### PR DESCRIPTION
**Pull request by oteg006**
### Description

add information form for collecting company info
impl #9 (issue numbers)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible

### CODE REVIEW

File: backend/api/company/company.go [hunk 1]
```diff
@@ -78,3 +78,50 @@ func CompanyGet(ctx context.ApplicationContext) gin.HandlerFunc {
 	}
 
 }
+
+const (
```
Comment (yuluo-yx) [inline backend/api/company/company.go:82]: move to `pkg` package, to public use in code files.

File: backend/api/company/company.go [hunk 2]
```diff
@@ -78,3 +78,50 @@ func CompanyGet(ctx context.ApplicationContext) gin.HandlerFunc {
...
+					Code: AddFailed,
+					Msg:  err.Error(),
+					Data: nil,
+				}))
+			}
+			return
+		}
+		c.AbortWithStatusJSON(http.StatusOK, types.NewValidResponse(&Resp{
+			Code: AddSuccess,
+			Msg:  "add info success",
```
Comment (yuluo-yx) [inline backend/api/company/company.go:122]: Try to avoid magic strings.

File: backend/dao/company/companyinfo.go [hunk 3]
```diff
@@ -2,11 +2,17 @@ package company
```
Comment (yuluo-yx) [inline backend/dao/company/companyinfo.go:2]: This file name should to changed `company_info` or `company`.

File: backend/dao/company/companyinfo.go [hunk 4]
```diff
@@ -2,11 +2,17 @@ package company
 
 import (
 	"context"
+
+	"github.com/pkg/errors"
```
Comment (yuluo-yx) [inline backend/dao/company/companyinfo.go:6]: Adjust package order.

File: backend/service/company/company_test.go [hunk 5]
```diff
@@ -2,15 +2,16 @@ package company
 
 import (
 	"context"
+  "regexp"
```
Comment (yuluo-yx) [inline backend/service/company/company_test.go:5]: Adjust code style.

**Review by yuluo-yx — CHANGES_REQUESTED** [review]
Please adjust code file by comment.

File: backend/service/company/company_test.go (hunk 5 above)
Comment (oteg006) [reply backend/service/company/company_test.go:5]: ok

File: backend/dao/company/companyinfo.go (hunk 4 above)
Comment (oteg006) [reply backend/dao/company/companyinfo.go:6]: ok

File: backend/dao/company/companyinfo.go (hunk 3 above)
Comment (oteg006) [reply backend/dao/company/companyinfo.go:2]: ok

File: backend/api/company/company.go (hunk 2 above)
Comment (oteg006) [reply backend/api/company/company.go]: ok

File: backend/api/company/company.go (hunk 1 above)
Comment (oteg006) [reply backend/api/company/company.go]: ok

File: backend/api/company/company.go [hunk 6]
```diff
@@ -1,6 +1,7 @@
 package company
 
 import (
+	"github.com/deamgo/uipaas-home/backend/pkg/e"
```
Comment (yuluo-yx) [inline backend/api/company/company.go:4]: pls sort

**Review by yuluo-yx — APPROVED** [review]
LGTM.

**Pull request merged**